packages/integrations/integrations.yml

```diff
@@ -26,7 +26,7 @@
 - name: Intercom
   ios:
     pod:
-      version: 1.0.0-beta
```

Comment:
do we need to upgrade these now that we are still on 3.+ series?

Reply:
No, since I back ported the iOS bits to 3.9, no changes necessary to the integrations.